lib/Runtime/Library/amd64/JavascriptFunctionA.S

```diff
@@ -108,38 +108,38 @@ NESTED_END amd64_CallFunction, _TEXT
 
 
 .balign 16
-.text
-C_FUNC(_ZN2Js18JavascriptFunction20DeferredParsingThunkEPNS_16RecyclableObjectENS_8CallInfoEz):
-        push rbp
+NESTED_ENTRY _ZN2Js18JavascriptFunction20DeferredParsingThunkEPNS_16RecyclableObjectENS_8CallInfoEz, _TEXT, NoHandler
+        push_nonvol_reg rbp
         lea  rbp, [rsp]
 
         // save argument registers used by custom calling convention
-        push rdi
-        push rsi
+        push_register rdi
+        push_register rsi
 
         // Call
         //  JavascriptMethod JavascriptFunction::DeferredParse(ScriptFunction**)
         //
         lea rdi, [rbp + 10h]    // &function, setup by custom calling convention
         call C_FUNC(_ZN2Js18JavascriptFunction13DeferredParseEPPNS_14ScriptFunctionE)
 
-        pop rsi
-        pop rdi
-        pop rbp
+        pop_register rsi
+        pop_register rdi
+        pop_nonvol_reg rbp
+NESTED_END _ZN2Js18JavascriptFunction20DeferredParsingThunkEPNS_16RecyclableObjectENS_8CallInfoEz, _TEXT
 
         jmp rax
```

Comment:
@obastemur Thanks for looking! Do you mean moving `jmp rax`? (Was troubled by this one. Would look fine if it were a `call` and `ret` before NESTED_END. But it is a jmp.)